src/main/resources/templates/company.html

```diff
@@ -1,51 +1,46 @@
 <!DOCTYPE html>
-<html xml:th="http://www.thymeleaf.org">
+<html xmlns:th="http://www.thymeleaf.org" lang="en-US">
 <head>
-  <link rel="stylesheet" href="index.css">
+  <title>Voluntier</title>
+  <link rel="stylesheet" th:href="@{css/Index.css}">
+
   <script>
     function openForm() {
       document.getElementById("myForm1").style.display = "block";
+      document.getElementById("signUpButton").style.display = "none"
     }
 
     function closeForm() {
-      document.getElementById("myForm").style.display = "none";
+      document.getElementById("myForm1").style.display = "none";
+      document.getElementById("signUpButton").style.display = "block"
```

Comment:
Updated JavaScript to hide/show newly added sign up button depending on if related form is showing.